tools/bin/commands/dev

```diff
@@ -128,5 +129,7 @@ dev::run() {
         echo $cookie_cached
     elif [ $(hasflag --version) ]; then
         oc get secret syndesis-global-config -o jsonpath={.data.syndesis} | base64 --decode
+    elif [ $(hasflag --clense ) ]; then
+        oc delete pods $(oc get pods --field-selector=status.phase=Succeeded -o=jsonpath='{range .items[*]}{.metadata.name} ')
```

Comment:
would remove the extraneous space at the end of the jsonpath expression

Reply:
Alternative you could also just use

```
oc delete $(oc get pods --field-selector=status.phase=Succeeded -o name)
```

(though not tested, but should work)

Reply:
Extra space is needed to separate pod names from each other, they get concatenated one to each other otherwise.

`-o name` would yield `pod/name` in the output where we need just the `name`, I actually quite dislike that it does that.